ui/src/pages/Logs/Logs.tsx

```diff
@@ -175,9 +230,25 @@ const logsDisplayStyle = {
   overflow: 'hidden',
   fontFamily: 'monospace',
 };
-function Row({ containerName, containerId, time, stream, log }: DockerLog) {
+
+// let containerIconColor: string;
+
+function Row({
+  containerName,
+  containerId,
+  time,
+  stream,
+  log,
+  containerLabelColor,
+  containerIconColor,
+}: DockerLog) {
   const [open, setOpen] = useState<boolean>(false);
 
+  const labelColor: string = containerLabelColor[containerId];
+  let iconColor: string;
+
+  containerIconColor[containerId] === 'running' ? (iconColor = 'teal') : (iconColor = 'grey');
```

Comment:
This technically works, but usually this would be written like:
`let iconColor = containerIconColor[containerId] === 'running' ?  'teal' : 'grey'`

Reply:
Similar as the comment above, you could also put this directly in the JSX instead of creating a new variable.

However, it is a style/readability decision. Both ways work! It can totally be argued that defining them here is more readable. :)